docs/pipelines/shiva.md

```diff
@@ -130,6 +135,14 @@ For all the options, please see the corresponding documentation for the mapping
 
 ## Advanced usage
 
+### Gender aware variantcalling
+
+In Shiva and ShivaVariantcalling while using haplotypecaller_gvcf it is possible to do gender aware variantcalling. In this mode it required to supply bed files to define haploid regions (see config values). 
+- For males is `hap̦loid_regions` and `hap̦loid_regions_male` used.
```

Comment:
Small nitpick on grammar: `for <gender>, haploid_regions and haploid_regions_<gender> is used`